treq/api.py

```diff
@@ -106,9 +109,10 @@ def request(method, url, **kwargs):
 #
 
 def _client(*args, **kwargs):
+    policy = kwargs.get('policy', BrowserLikePolicyForHTTPS())
```

Comment:
Although dropping support for earlier Twisteds is an acceptable thing to do (see #98 ) coupling to `BrowserLikePolicyForHTTPS` when it's not necessary seems like a bad way to implement this.  We don't actually use the `policy` object except to pass it through; we could simply not pass it and allow `Agent`'s default to take over if it's not passed to us.